docs/ide/default-keyboard-shortcuts-in-visual-studio.md

```diff
@@ -373,7 +373,7 @@ The sections in the following table include commands that are global in that you
 |View.ObjectBrowser|**Ctrl+Alt+J**|
 |View.ObjectBrowserGoToSearchCombo|**Ctrl+K, Ctrl+R**|
 |View.Output|**Ctrl+Alt+O** (letter 'O')|
-|View.PopBrowseContex|**Ctrl+Shift+8**|
+|View.PopBrowseContext|**Ctrl+Shift+8** (C++ Language Only)|
```

Comment:
```suggestion
|View.PopBrowseContext|**Ctrl+Shift+8** (C++ only)|
```

Reply:
I committed your suggestion. Now I'll resolve this conversation (I'm kinda new to github so let me know if there's anything else I should do here)